capstone/src/main/java/com/google/sps/servlets/AssociationServlet.java

```diff
@@ -35,15 +35,20 @@ public void init() throws ServletException {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(OUTPUT_TYPE);
 
+    String scope = request.getParameter("scope");
+    if (scope == null) {
```

Comment:
scope is never null because the precinct variable in the script file is initialized to "SF"

Reply:
I don't call it with null, but I still wanted the server request to have a built in default value for scope. I needed a way to handle requests that didn't include a scope parameter and chose this.